sdk/node/README.md

```diff
@@ -1,83 +1,308 @@
-# Hyperledger Client SDK for Node.js
+Hyperledger Client SDK for Node.js
+=========
+
+The Hyperledger Client (HLC) provides a powerful and easy to use API to interact with a Hyperledger blockchain.
```

Comment:
Need to call it the Hyperledger fabric client SDK, because calling it Hyperledger Client SDK is too general given that there are multiple incubating projects. This change needs to be made throughout.